eslint.config.mjs

```diff
@@ -177,9 +178,10 @@ export default [
 			"sonarjs/cognitive-complexity": "error",
 			"sonarjs/no-all-duplicated-branches": "error",
 			"sonarjs/no-collapsible-if": "error",
-			"sonarjs/no-duplicate-string": ["error", { "threshold": 5 }],
+			"sonarjs/no-duplicate-string": ["error", { threshold: 5 }],
 			"sonarjs/no-identical-expressions": "error",
 			"sonarjs/no-identical-functions": "error",
+			"sonarjs/no-one-iteration-loop": "off",
```

Comment:
let me recheck if this is still needed, don't think so after adjusting the uniq/uniq-by files